tests/components/minecraft_server/test_config_flow.py

```diff
@@ -185,10 +242,6 @@
                 )
 
                 assert result["type"] == RESULT_TYPE_CREATE_ENTRY
-                assert (
-                    result["title"]
-                    == f"{USER_INPUT_IPV6[CONF_HOST]}:{USER_INPUT_IPV6[CONF_PORT]}"
-                )
+                assert result["title"] == USER_INPUT_IPV6[CONF_HOST]
                 assert result["data"][CONF_NAME] == USER_INPUT_IPV6[CONF_NAME]
-                assert result["data"][CONF_HOST] == USER_INPUT_IPV6[CONF_HOST]
-                assert result["data"][CONF_PORT] == USER_INPUT_IPV6[CONF_PORT]
+                assert result["data"][CONF_HOST] == "::ffff:0101:0101"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5232399)